src/model/import.ts

```diff
@@ -59,6 +59,9 @@ export class ScannedImport implements Resolvable {
   }
 
   resolve(document: Document): Import|undefined {
+    if (!document.analyzer.canResolveUrl(this.url)) {
```

Comment:
... What's changed is that this makes it so we don't add warnings for remote/out-of-package imports.  The absence of ERROR severity warnings enables `polymer-build` to do the right thing and not pretend to resolve unloadable files with empty strings which is what it does erroneously right now.